### DISCUSSION TITLE
Cypress always open headless with npx cypress run

### DISCUSSION THREAD

**Discussion by mbarsocchi** [cypress-io/cypress#18038] [Questions and Help]
To run a single spec test case in a real browser, I've always used the command:

`npx cypress run  --browser chrome --spec "somespec.spec.js"`

But from today on, cypress always open headless with Chrome or Firefox, even if I've tried with

`--headless false`

Someone experiencing this issue?

Cypress package version: 8.3.0
Mac Book Big Sur 11.4
Node 16.6.1

**Comment by romankhomitskyi** [comment]
npx cypress open is for headed mode
